client/components/rating/index.jsx

```diff
@@ -9,7 +9,7 @@ import './style.scss';
 export default class Rating extends PureComponent {
 	static defaultProps = {
 		rating: 0,
-		size: 24,
+		size: 18,
```

Comment:
There is a test failing because of this change

> <Rating /> check props size should be set to 24px if no size

* The version with the feature flag turned off, which also uses this component, still "looking good" with this new size?
* If we still want to proceed with this new size, I would suggest adding a prop called `size` to `PluginRating` where we can set this value [on the old version](https://github.com/Automattic/wp-calypso/blob/f2e6f88ae65c8571bbac21b284c29b48ec987598/client/my-sites/plugins/plugin-information/index.jsx#L286).

```jsx
<PluginRatings
  rating={ this.props.plugin.rating }
  ratings={ this.props.plugin.ratings }
  downloaded={ this.props.plugin.downloaded }
  numRatings={ this.props.plugin.num_ratings }
  slug={ this.props.plugin.slug }
  size={ 18 }
/>
```

Reply:
Thanks for looking into the test failure 👏 !

> The version with the feature flag turned off, which also uses this component, still "looking good" with this new size?

My mistake for not mentioning it, yes I checked it and it is acceptable
<img width="795" alt="SS 2021-11-17 at 11 05 40" src="https://user-images.githubusercontent.com/12430020/142170244-52467adb-45f1-40aa-aab5-784b48425664.png">

> If we still want to proceed with this new size, I would suggest adding a prop called size to PluginRating where we can set this value on the old version.

I understand your point here but this code will become obsolete (and we would better also deprecate it) after launching Marketplace V0.5.

Reply:
> I understand your point here but this code will become obsolete (and we would better also deprecate it) after launching Marketplace V0.5.

Got it, ty for explaining.